src/sonic_ax_impl/mibs/ietf/rfc2863.py

```diff
@@ -182,6 +182,34 @@ def _get_counter(self, oid, table_name, mask):
             mibs.logger.warning("SyncD 'COUNTERS_DB' missing attribute '{}'.".format(e))
             return None
 
+    def _get_if_entry(self, sub_id):
+        """
+        :param oid: The 1-based sub-identifier query.
+        :return: the DB entry for the respective sub_id.
+        """
+        oid = self.get_oid(sub_id)
+        if not oid:
+            return
+
+        table = ""
```

Comment:
>table [](start = 8, length = 5)

table is not a good name.

Reply:
I did not write this function, I lifted it directly from rfc1213.py
if we are going to refactor existing utility code I feel it should be done in another pull request and work with the "refactor rfc1213.py and rfc2863.py" issue

Reply:
I did not intend to blame you or your code. Even it is copied from the old files, it is your responsibility to make it best at new places.

You may prefer don't touch unrelated code. I am ok. Just keep new code better.

---
In reply to: [207958146](https://github.com/Azure/sonic-snmpagent/pull/73#discussion_r207958146) [](ancestors = 207958146)